sdk/cognitivelanguage/ai-language-text/test/internal/warn_text.spec.ts

```diff
@@ -31,7 +31,6 @@ describe("Logging", function () {
     }
     setLogLevel("warning");
     await client.beginAnalyzeBatch([{ kind: "EntityRecognition" }], ["I need coffee"], "en");
-    sinon.assert.callCount(spy, 1);
```

Comment:
could you share why this is needed?

Reply:
I saw from [a previous run failure](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3963076&view=logs&j=95d26128-725d-5659-20da-21d48988bc39&t=e7bf5964-e52c-5bcb-8d69-6c065a55c903&l=2832) that the count includes logging received from `core-lro`. There is validation that we receive the warning log, so I remove this line.